js/history.js

```diff
@@ -126,6 +192,7 @@ Fliplet.Registry.set('history-stack', (function() {
     add: add,
     back: back,
     forward: forward,
-    getCurrent: getCurrent
+    getCurrent: getCurrent,
+    columnsInfo,
```

Comment:
It doesn't seem appropriate to be embedding a `columnsInfo` object into `HistoryStack` with its own methods when the majority of what's in `columnsInfo` doesn't seem to be interacting with the history stack.

I would suggest using a separate class `DSColumn` in a separate file to do this.

I do see that `original()` does use `stack[0].data[0]`, which references `stack` but that could be done through a new or existing public method in `HistoryStack`.